src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1251,14 +1251,59 @@ void TwitchMessageBuilder::appendTwitchBadges()
                 break;
             }
         }
+        else if (badge.startsWith("founder/"))
+        {
+            if (auto badgeEmote =
+                    this->twitchChannel->globalTwitchBadges().badge("founder",
+                                                                    "0"))
+            {
+                auto badgeInfo = this->tags.find("badge-info");
+                if (badgeInfo != this->tags.end() &&
+                    badgeInfo.value().toString().split(',')[0].startsWith(
+                        "founder/"))
+                {
+                    auto subMonths =
+                        badgeInfo.value().toString().split(',')[0].mid(8);
```

Comment:
I think it would be more flexible to move the `"founder/"` to variable and use a `founderStr.size()` instead of a raw number `8`.

The same thoughts about `"subscriber/"` and `mid(11)`.